x/bank/client/rest/sendtx.go

```diff
@@ -16,7 +16,7 @@ import (
 
 // RegisterRoutes - Central function to define routes that get registered by the main application
 func RegisterRoutes(cliCtx context.CLIContext, r *mux.Router, cdc *codec.Codec, kb keys.Keybase) {
-	r.HandleFunc("/accounts/{address}/send", SendRequestHandlerFn(cdc, kb, cliCtx)).Methods("POST")
+	r.HandleFunc("/bank/accounts/{address}/transfers", SendRequestHandlerFn(cdc, kb, cliCtx)).Methods("POST")
```

Comment:
Why `transfers` when the endpoint's purpose is to `send` coins?

Reply:
I modify this endpoint according to this [issue](https://github.com/cosmos/cosmos-sdk/issues/2113)

Reply:
@alessio `POST /transfers` is more restful then to use an RPC endpoint like `/send` and we try to make the API consistent. Do you think it is too confusing?

Reply:
No, I actually agree on the name's restfulness

Reply:
So what is the decision of this issue?

Reply:
let's keep it